tests/andromede/test_xpansion.py

```diff
@@ -76,12 +95,64 @@ def thermal_candidate() -> Model:
                 name="Max generation", expression=var("generation") <= var("p_max")
             )
         ],
-        objective_contribution=(param("invest_cost") * var("p_max"))
-        + (param("op_cost") * var("generation")).sum().expec(),
+        objective_operational_contribution=(param("op_cost") * var("generation"))
+        .sum()
+        .expec(),
+        objective_investment_contribution=param("invest_cost") * var("p_max"),
     )
     return THERMAL_CANDIDATE
 
 
+@pytest.fixture
+def wind_cluster_candidate() -> Model:
+    WIND_CLUSTER_CANDIDATE = model(
```

Comment:
Rename this candidate as `DISCRETE_CANDIDATE` which is more explicit regarding its behaviour and also because usually wind is considered a continuous candidate (whereas gas plant are not for instance)

Reply:
Done